differs/pipDiff.go

```diff
@@ -37,27 +39,50 @@ func getPythonVersion(pathToLayer string) ([]string, error) {
 	return matches, nil
 }
 
-func (d PipDiffer) getPackages(path string) (map[string]map[string]utils.PackageInfo, error) {
-	packages := make(map[string]map[string]utils.PackageInfo)
+func getPythonPaths(vars []string) []string {
```

Comment:
What do you mean the path set on exit? 
But also, now that you mention it, I've been obtaining packages from the PYTHONPATH in addition to the default "site-packages" path, but perhaps this is wrong...  would no packages be installed in that default location if the user explicitly sets their PYTHONPATH?